recipes/catkin/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: ros-catkin
+  version: 0.7.17
+
+source:
+  - url: https://github.com/ros-gbp/catkin-release/archive/release/melodic/catkin/0.7.17-0.tar.gz
+    fn: ros-catkin-0.7.17.tar.gz
+    sha256: 316178707bc3f0edcb86a77ec2ec174b98b4d2ccf41306162cefe6588b3a9d41
+    patches:
+      - 001_remove_rt.patch
+      - 002_no_hardcoded_python.patch
+  - path: 003_ros_setup.patch
```

Comment:
These path lines aren't needed are they? You can use `${RECIPE_DIR}/003_ros_setup.patch` instead
```suggestion
```

Reply:
ah cool, didn't know about this trick!